### DISCUSSION TITLE
Morocco option doesn't exist in 2FA SMS

### DISCUSSION THREAD

**Discussion by x3n0n21** [community/community#130621] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, i was trying to set up 2FA by SMS but couldn't find the Morocco(+212) region among the options provided 
![Capture d'écran 2024-07-01 133233](https://github.com/community/community/assets/174336220/28429e69-9abe-4086-8b69-99c6ce91b514)

**Comment by ebndev** [comment, marked as answer]
Hey there @x3n0n21,

Thanks so much for reaching out in GitHub's Community! If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
